osidb/serializer.py

```diff
@@ -861,6 +862,73 @@ class Meta:
             + TrackingMixinSerializer.Meta.fields
         )
 
+    def update(self, affect, validated_data):
+        """
+        perform the tracker instance update
+        """
+
+        # NOTE: This overrides update() behavior provided by mixins, but mostly
+        #       mirrors the mixins. If mixins change, this method must be updated.
+        # NOTE: Look at TrackerSerializer.update() and
+        #       FlawPackageVersionSerializerMixin.update() for similar ways of
+        #       interposing actions within the mixins' update() MRO & flow.
+
+        ############################
+        # 1) prepare prerequisites #
+        ############################
+
+        # From ACLMixinSerializer.update()
+        validated_data = self.embargoed2acls(validated_data)
+
+        #########################
+        # 2) pre-update actions #
+        #########################
+
+        old_affectedness = affect.affectedness
+
+        #####################
+        # 3) update actions #
+        #####################
+
+        # From BugzillaSyncMixinSerializer.update()
+        for attr, value in validated_data.items():
+            setattr(affect, attr, value)
+
+        # From BugzillaBareSyncMixinSerializer.update()
+        affect.save(bz_api_key=self.get_bz_api_key())
+
+        ##########################
+        # 4) post-update actions #
+        ##########################
+
+        # Trigger state-based tracker save to regenerate validation-requested label.
+        # If a tracker is New and if all its affects are NEW, tracker must have label validation-requested.
+        # Otherwise, the label must not be present. Therefore, re-save trackers when affectedness
+        # changes to/from NEW.
+
+        if (
+            old_affectedness != affect.affectedness
+            and Affect.AffectAffectedness.NEW in [old_affectedness, affect.affectedness]
+        ):
+            # Status values governed by Jira project workflow, not accessible through JiraProjectFields.
+            # Most probably, New will always be something like NEW, New, new.
+            for tracker in affect.trackers.filter(
+                status__in=["NEW", "New", "new"]
```

Comment:
Again I do not think that we should care for the tracker status.

Reply:
Removed in today's commit. Please re-review.